server/src/controller/order.controller.ts

```diff
@@ -36,33 +34,70 @@ export const postOrder = async (req: Request, res: Response) => {
 
 
 export const getOrders = async (req: Request, res: Response) => {
-	const orders = await db.Order.findAll({
-		attributes: ['id', 'startWorkOn', 'endWorkOn'],
-		include: [
-			{
-				model: db.Clock,
-				attributes: ['id', 'size'],
-				required: true,
-			},
-			{
-				model: db.User,
-				attributes: ['id', 'name', 'email'],
-				required: true,
-			},
-			{
-				model: db.City,
-				attributes: ['id', 'name'],
-				required: true,
-			},
-			{
-				model: db.Master,
-				attributes: ['id', 'name'],
-				required: true,
-			},
-		],
-	});
+	try {
+		const token = (<string>req.headers.authorization).split(' ')[1];
+
+		const {id: userId, role: userRole, masterId: masterId} = await db.User.findOne({where: {token}});
+
+		if (userRole === 'admin') {
+			const orders = await db.Order.findAll({
```

Comment:
if I see the difference correctly, you have the order parameter on the master role. the other stuff is same. Why so big if instead of smth like isAdmin && order: {}